### PR TITLE
feat: Added message for Holopin related questions

### PR DESCRIPTION
**Pull request by polarvoid**
# Pull Request

## Description:

Adds a small message for queries regarding Holopin. I see a ton of questions on the HF server that are one of the following:
- Where are my Holopins?
- Why are my Holopins not showing up?

and quite often, they have either not opted-in to it, or it's in their spam :/

### CODE REVIEW

**Comment by polarvoid** [comment]
Hey @naomi-lgbt I also noticed the `CONTRIBUTING` guidelines are outdated (I think?) with regard to the way the tags work currently with the '/' system.

**Comment by polarvoid** [comment]
I just saw that there's a new `Claim it now` link too, maybe this could include that as well?

File: src/data/tags.ts [hunk 1]
```diff
@@ -118,4 +118,9 @@ export const tags: Tag[] = [
       "Hacktoberfest doesn't have any winners or losers. Positive contributions to the open-source community is a win for everyone",
     question: "Can I/how do I win Hacktoberfest?",
   },
+  {
+    answer:
+      "You have to opt-in via email for Holopin badges. Check your spam/trash before asking again.",
```
Comment (naomi-lgbt) [inline src/data/tags.ts:123]:
```suggestion
      "You have to opt-in via email for Holopin badges. If you opted in, make sure that the email did not land in your spam folder. Also, be sure that you opted in with the correct email address (not the GitHub no-reply email address). Finally, you should also see a `Claim Now` link in your Hacktoberfest profile.",
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Let's soften the tone a bit and provide some additional information.

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Comment by polarvoid** [comment]
Awesome, thanks Naomi :)

**Pull request merged**